test/unit/serializer.cpp

```diff
@@ -81,11 +81,16 @@ struct serializer_test
     {
         std::string s;
         auto cbs = sr.prepare().value();
-        s.resize(buffers::buffer_size(cbs));
+        // we limit buffer consumption to complicate
+        // the serializer's internal buffer management
+        auto consumed = std::min(
```

Comment:
This has to be protected with parenthesis to guard against macros, e.g. `(std::min)(...`